src/Dynamo.All.Net6.sln

```diff
@@ -864,6 +867,22 @@ Global
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|NET60_Windows.Build.0 = Release|NET60_Windows
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.ActiveCfg = Release|x64
 		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.Build.0 = Release|x64
+		{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I've been meaning to remove these other platforms (x64,anycpu) I guess it doesn't get harder the longer we wait with the configuration manager ui.

Reply:
I;ve been meaning to do the same